osu.Game.Tests/Visual/Menus/TestSceneToolbarClock.cs

```diff
@@ -76,5 +88,37 @@ public void TestLongGameTime()
         {
             AddStep("Set game time long", () => mainContainer.Clock = new FramedOffsetClock(Clock, false) { Offset = 3600.0 * 24 * 1000 * 98 });
         }
+
+        [Test]
+        public void TestHoverBackground()
```

Comment:
This test as a whole is maybe... too much? For something like a hover background we expect it to work and adding too many tests like this would generate noise.

Reply:
This was meant to be "my first test" to get used to writing these, I'll remove it.